packages/e2e-test-utils/src/index.js

```diff
@@ -35,6 +35,7 @@ export {
 	insertBlockDirectoryBlock,
 	openGlobalBlockInserter,
 	closeGlobalBlockInserter,
+	toggleGlobalBlockInserter,
```

Comment:
There are 2 existing helpers to control the block inserter. Would it be possible to use them instead of introducing another helper function?

Reply:
👋🏻 Hi Greg! I tried `openGlobalBlockInserter` but it waits for the actual inserter el to be present on the page. In the case the error ever regresses, it would cause a timeout because the inserter is never shown. What we really need is to just click to show the inserter. What's the other helper?

Reply:
Also, this is actually an existing helper, I just exported it.

Reply:
> I tried `openGlobalBlockInserter` but it waits for the actual inserter el to be present on the page. In the case the error ever regresses, it would cause a timeout because the inserter is never shown.

It looks like `openGlobalBlockInserter` is exactly what fits best here. `toggleGlobalBlockInserter` only verifies if the button can be clicked. The waiting part is crucial here to prevent regression. If it would timeout in the future, that's a proper way to signal that something went wrong.

Reply:
> toggleGlobalBlockInserter only verifies if the button can be clicked.

Hmm, it actually wraps a `click` call: https://github.com/WordPress/gutenberg/blob/master/packages/e2e-test-utils/src/inserter.js#L49. For the purposes of this regression test, that's all we need.

If we decide to use `openGlobalBlockInserter`, there's another potential problem as well (which I just tested): if it the bug regresses, it will fail with the timeout and not with the expect below it. It also means it will slow down the actual test that would finish faster if we weren't waiting for an inserter we know won't appear.